packages/shared/src/components/fields/TextField.module.css

```diff
@@ -13,3 +13,9 @@
     border-color: var(--theme-status-error);
   }
 }
+
+.field input[type='number']::-webkit-inner-spin-button,
+.field input[type='number']::-webkit-outer-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
```

Comment:
Why do we need to set the margin to 0?
It messes the design as I saw on your print screen

Reply:
I see. It was mentioned in the issue I saw that there might be some unwanted margins due to the spin button we've hidden.

After removing the `margin: 0`, I didn't notice any changes though, so it's safe to say I can remove this then, which I will now.